QUARKUS-5858.md

```diff
@@ -0,0 +1,30 @@
+# QUARKUS-5858 Tech preview Support for OAuth2 Demonstrating Proof of Possession
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5858
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/45891
+
+Upstream documentation:
+* https://quarkus.io/version/main/guides/security-oidc-bearer-token-authentication#demonstrating-proof-of-possession-dpop
+
+## Preview status
+DPoP is currently considered tech-preview in quarkus.
+
+## Scope of testing
+Verify quarkus works with authentication method "Demonstrating Proof of Possession" (DPoP)
+A new test class will be implemented in `security/keycloak-oidc-client-extended`.
+Tests will verify authentication using DPoP, which will cover test cases:
+* Happy path - correctly accessing resource using DPoP authentication.
+* Authentication failures:
+  * No DPoP header.
+  * DPoP proof signed with wrong key.
+  * Mismatching HTTP method in DPoP proof.
+  * Mismatching HTTP path in DPoP proof.
+
+### Impact on resources
+A new module will be created, requires additional test execution.
+Tests will be executed on both JVM and native and on both baremetal and OCP.
+Expected additional execution time in minutes for each case.
```

Comment:
I expect minutes per each test class, so if you plan to have case per class, you should mention how many test cases you expect. Otherwise without saying how many classes or cases  you are adding, we don't know how much you add minutes.

Reply:
In this context by "each case" I mean - few minutes for JVM bare metal + few minutes for OCP native .... So in total 4times "few minutes".
I can break it down into all the cases if you want.

Reply:
I am ok with it now that I understand what you meant